packages/tessellate-request/lib/configuration/oauth-configuration-file-loader.js

```diff
@@ -0,0 +1,16 @@
+// @flow
+
+import nconf from 'nconf'
+import fs from 'fs'
+import type {OAuthConfig} from './oauth-configuration'
+
+let configurations: Array<OAuthConfig>
+
+export function init() {
```

Comment:
It's not a good idea to rely on clients to call the `init()` function first. Otherwise `configurations` would be `undefined` and this also means that `get()`'s type is actually `() => ?Array<OAuthConfig>`.

Also, clients need to implement the lookup by id manually every time. It would be cool if this module would encapsulate:
- lazy loading of `OAUTH_CONFIGURATIONS_FILE`
- lookup by id

So its API could be simply:
```javascript
export default get(id: string): OAuthConfig { }
```
That way we can also easily change the implementation of the configuration lookup.